src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -194,6 +196,115 @@ tags:
       }
       ```
 
+      ## Consistent reads
+
+      As described in [Browsing historical data](#section/Browsing-historical-data) in all Network Gateway requests it is possible to specify the exact ledger state by providing the `at_ledger_state` parameter.
+
+      It can be used to browse historical data but it also allows consistent reads when making multiple requests (either when paginating or when collecting data from multiple endpoints).
+
+      Imagine a situation when your key value store contains 128 keys. To fetch all the keys and it is data you would need to make at least 4 requests (2 to paginate through all keys and 2 to fetch all the data). Using the same `at_ledger_state` parameter value it is guaranteed that you are fetching all the data from exactly the same ledger state.
+
+      Example requests:
+
+      1st page of keys:
+      ```
+      /state/key-value-store/keys
+
+      {
+        "key_value_store_address": "internal_keyvaluestore_rdx1kprq4hx5892fuezay6tyajyu0gr2jeu8j7alauxen46xu8ufzvecph"
+      }
+      ```
+
+      Response:
+      ```
+      {
+        "ledger_state": {
+          "network": "mainnet",
+          "state_version": 50576725,
+          "proposer_round_timestamp": "2024-02-20T16:46:22.343Z",
+          "epoch": 74324,
+          "round": 84
```

Comment:
Just thinking out loud here. I'd trim some of those unimportant response properties (`netwrok, timestamp, epoch, round`) and leave only those that are important to understand the concept. Just to reduce the noise in those examples. Similarly to how we `...` all the `items`.